openshift/4.0/templates/jenkins-slaves/node10-jenkins-slave.yaml

```diff
@@ -0,0 +1,39 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: nodejs-10-jenkins-slave
+  labels:
+    role: jenkins-slave
+  namespace: 354028-tools
+data:
+  node10: |-
+    <org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
```

Comment:
Odd to have Alejandro's namespace here

Reply:
https://github.com/jenkinsci/kubernetes-plugin